src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -351,6 +352,10 @@ public PdaoLoadStatistics loadToStagingTable(Dataset dataset,
         }
 
         if (loadJob.getStatus().getError() != null) {
+            if ("notFound".equals(loadJob.getStatus().getError().getReason())) {
```

Comment:
nit: do we have an enum for this?

Reply:
Unfortunately not... We could make one based on [this table](https://cloud.google.com/bigquery/troubleshooting-errors), but it wouldn't give us any extra protection against the strings changing out from under us.

If this is too magic string-y I can remove it.

Reply:
I think it is useful as is.